FoursquareTopTests/VenueDetailViewControllerDriver.swift

```diff
@@ -29,4 +31,63 @@ class VenueDetailViewControllerDriver {
             tester.waitForAbsenceOfViewWithAccessibilityLabel(categoryName)
         }
     }
-}
\ No newline at end of file
+    
+    func expectMenuButton(toBeVisible visible: Bool) {
+        if visible {
+            testCase.waitForViewWithLocalizedAccessibilityLabel(.VenueDetailSeeMenu)
+        } else {
+            testCase.waitForAbsenceOfViewWithLocalizedAccessibilityLabel(.VenueDetailSeeMenu)
+        }
+    }
+    
+    func expectAddressLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool) {
+        
+        expectView(withText: venue.address, withAccessibilityLabel: tr(.VenueDetailAddressAccessibilityLabel), toBeVisible: visible)
+        
+    }
+    
+    func expectStatusLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool) {
+        
+        expectView(withText: venue.status, withAccessibilityLabel: tr(.VenueDetailStatusAccessibilityLabel), toBeVisible: visible)
+      
+    }
+    
+    func expectRatingLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool) {
+        
+        expectView(withText: venue.formattedRating, withAccessibilityLabel: tr(.VenueDetailRatingAccessibilityLabel), toBeVisible: visible)
+        
+    }
+    
+    func expectPricingLabel(ofVenue venue: VenueViewModel, toBeVisible visible: Bool)
+    {
+        expectView(withText: venue.formattedPrice, withAccessibilityLabel: tr(.VenueDetailPricingAccessibilityLabel), toBeVisible: visible)
+    }
+    
+    private func expectView(withText text: String? = nil, withAccessibilityLabel label: String? = nil, toBeVisible visible: Bool) {
+        
+        if let accessibilityLabel = label {
+            if visible {
+                let view = testCase.waitForViewWithLocalizedAccessibilityLabel(accessibilityLabel) as UIView
+                
+                if let label = view as? UILabel {
+                    failIfNotEqual(label.text!, expected: text!)
```

Comment:
I think you can write this using the expectations framework, maybe something like

```
expect(label.text!).to(be(text))
```

or something similar (can't remember the syntax, sorry)